packages/paste-core/components/pagination/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
```

Comment:
I split customization tests into its own file because the main test file is huge.